packages/code-du-travail-frontend/src/outils/common/ShowDetails.tsx

```diff
@@ -1,19 +1,68 @@
-import { Accordion } from "@socialgouv/cdtn-ui";
+import { AccordionArrow, Text } from "@socialgouv/cdtn-ui";
 import React from "react";
+import styled from "styled-components";
+
+import { matopush } from "../../piwik";
 
 type Props = {
   children: React.ReactNode;
 };
 
-const ShowDetails = ({ children }: Props): JSX.Element => (
-  <Accordion
-    items={[
-      {
-        body: children,
-        title: <p>Voir le détail du calcul</p>,
-      },
-    ]}
-  />
-);
+const ShowDetails = ({ children }: Props): JSX.Element => {
+  const [active, setActive] = React.useState(false);
+  const [height, setHeight] = React.useState("0px");
+  const contentSpace = React.useRef(null);
+
+  function toggleAccordion() {
+    setActive(!active);
+    setHeight(!active ? `${contentSpace.current.scrollHeight}px` : "0");
+    matopush(["TRACK_EVENT", "OUTIL", ""]); // TODO replace with events
```

Comment:
Wait for branch : https://github.com/SocialGouv/code-du-travail-numerique/pull/3864

{
    "eventCategory": "outil",
    "eventAction": "Préavis de Retraite",
    "eventName": "click_calcul_détail" => **pas de dédoublement**
}